server/game/cards/03_TWI/leaders/MaulARivalInDarkness.ts

```diff
@@ -0,0 +1,35 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { LeaderUnitCard } from '../../../core/card/LeaderUnitCard';
+import { KeywordName } from '../../../core/Constants';
+
+export default class MaulARivalInDarkness extends LeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '6461101372',
+            internalName: 'maul#a-rival-in-darkness',
+        };
+    }
+
+    protected override setupLeaderSideAbilities() {
+        this.addActionAbility({
+            title: 'Attack with a unit. It gains Overwhelm for this attack',
+            cost: [AbilityHelper.costs.exhaustSelf()],
+            initiateAttack: {
+                attackerCondition: (card, context) => card.controller === context.source.controller,
```

Comment:
This isn't needed, by default you can only initiate an attack with your own units